README.md

```diff
@@ -78,14 +78,7 @@ The following instructions assume a clean environment and show how to install PH
 
```

Comment:
Please update the introduction paragraph in line 74 `Install (UNIX)` as well
Another line to update: line 56 (remove the term `RC`) please